spec/bookingsync/api/client/contacts_spec.rb

```diff
@@ -0,0 +1,103 @@
+require "spec_helper"
+
+describe BookingSync::API::Client::Contacts do
+  let(:api) { BookingSync::API::Client.new(test_access_token) }
+
+  before { |ex| @casette_base_path = casette_path(casette_dir, ex.metadata) }
+
+  describe ".contacts", :vcr do
+    it "returns contacts" do
+      expect(api.contacts).not_to be_empty
```

Comment:
can we extract the subject of the spec to a named `subject` like `subject(:contacts) { api.contacts }` same thing for all the specs

Reply:
`api.contacts` it's only used in this test. For show we use `api.contact`,  for edit `api.edit_contact` and for delete `api.delete_contact`. Shouldn't be better to just call method instead of using one named subject for each one?

Reply:
for every method, there will be a different subject. What I meant is to start using `subject` instead of having it as it is now. This is what we have in our styleguide for specs